source/documentation/standards/code-freezes.html.md.erb

```diff
@@ -0,0 +1,133 @@
+---
+owner_slack: "#operations-engineering-alerts"
```

Comment:
I wonder if this should actually be `#hosting`? 🤔

Reply:
It can be, it means some one in hosting will need to review and update the document at each review date expiry